cmd/clusterctl/client/config/providers_client.go

```diff
@@ -45,6 +45,7 @@ const (
 	DOProviderName             = "digitalocean"
 	GCPProviderName            = "gcp"
 	HetznerProviderName        = "hetzner"
+	HivelocityProviderName     = "hivelocity"
```

Comment:
To follow https://main.cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#adding-a-provider-to-clusterctl : the provider should contain the github org name so if I'm right in this case it should be `hivelocity-hivelocity` .

Reply:
The test is failing after adding the prefix. 

From docs linked above: 
_The need to add a prefix for providers not in the kubernetes-sigs org applies to all the providers being added to clusterctl‘s pre-defined list of provider starting from January 2024._

After reading the docs, I don't have full clarity on what the prefix should be?

Reply:
@chrischdi After I migrated the PR to use the new naming convention suggested above, the tests are failing, it was not failing when I used only `hivelocity` as provider name. Is there something else I should be aware of? 
Also is the prefix `hivelocity` correct in this case or this should be something else?

Reply:
My guess is: the tests were not touched yet when introducing the new naming schema, this is the first provider which gets added with the new schema I guess.

Reply:
I rebased the PR against main and then updated as per the above PR. The tests are failing with the following message. 

```bash
Gomega truncated this representation as it exceeds 'format.MaxLength'.
        Consider having the object provide a custom 'GomegaStringer' representation
        or adjust the parameters in Gomega's 'format' package.
        
        Learn more here: https://onsi.github.io/gomega/#adjusting-output
```
I tested this on main branch (cd cmd/clusterctl/cmd/; go test -run Test_runGetRepositories) and on main this error is not there.